mqt/bench/utils/utils.py

```diff
@@ -308,55 +310,40 @@ def get_molecule(benchmark_instance_name: str):
     return instances[benchmark_instance_name]
 
 
-def postprocess_oqc_qasm_files():
-    directory = get_qasm_output_path()
-    Parallel(n_jobs=-1, verbose=100)(
-        delayed(postprocess_single_oqc_file)(directory, filename)
-        for filename in os.listdir(directory)
-    )
-
+# def postprocess_oqc_qasm_files():
+#     directory = get_qasm_output_path()
+#     Parallel(n_jobs=-1, verbose=100)(
+#         delayed(postprocess_single_oqc_file)(directory, filename)
+#         for filename in os.listdir(directory)
+#     )
```

Comment:
if this is no longer needed, then it might as well be removed.

Reply:
Done.